.github/workflows/main.yml

```diff
@@ -15,12 +15,12 @@ jobs:
     - run: git fetch origin master
     - uses: flux-framework/pr-validator@master
 
-
-  run-checks:
+  check-python-formatting:
+    name: check python formatting
```

Comment:
It appears this job runs more than just python formatting (I'm not sure exactly what kind of style `flake8` enforces though, so apologies if I'm wrong), so maybe a more appropriate name would `python style and format`?

Also, I didn't see it, but are you running `pylint` as part of CI?

Reply:
> It appears this job runs more than just python formatting (I'm not sure exactly what kind of style `flake8` enforces though, so apologies if I'm wrong), so maybe a more appropriate name would be `python style and format`?

That seems like a more appropriate name - I can change it. Thanks for the feedback! 

> Also, I didn't see it, but are you running `pylint` as part of CI?

I am not running `pylint` as of now, but I can add it.

Reply:
Ah, sorry for the noise - I didn't know what `pylint` was but I just looked it up. Looks like it is also a style checker? Which is, to my understanding, very similar to what `flake8` does.

Reply:
Ah, maybe @SteVwonder can comment since I'm not really familiar with the difference. However, in my mind it would be nice if we used the same version of pylint between flux-framework projects that use Python to ensure a kind of uniformity. However, that is just my opinion and I'm willing to go with whatever you and @SteVwonder decide.

Reply:
I don't really have a preference for pylint vs flake8 in general, but I agree with @grondo that it would be good to keep things consistent across flux-framework projects.